### PR TITLE
docs(fr): correction of nested routes and components

### PR DESCRIPTION
**Pull request by ugocottin**
Posts route did not match with the above documentation route path.
UserHome and UserProfile were used for the same route, correction of UserHome to UserProfile

### CODE REVIEW

File: docs/fr/guide/essentials/nested-routes.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ const router = new VueRouter({
       children: [
         // `UserProfile` va être rendu à l'intérieur du `<router-view>` de `User`
         // quand `/utilisateur/:id` concorde
-        { path: '', component: UserHome },
+        { path: '', component: UserProfile },
```
Comment (posva) [inline docs/fr/guide/essentials/nested-routes.md]: It's the other way around, `UserProfile` should be `UserHome`
```suggestion
        { path: '', component: UserHome },
```

File: docs/fr/guide/essentials/nested-routes.md [hunk 2]
```diff
@@ -57,13 +57,13 @@ const router = new VueRouter({
...
           // `UserProfile` va être rendu à l'intérieur du `<router-view>` de `User`
-          // quand `/utilisateur/:id/profil` concorde
+          // quand `/utilisateur/:id/profile` concorde
           path: 'profile',
           component: UserProfile
         },
         {
           // `UserPosts` va être rendu à l'intérieur du `<router-view>` de `User`
-          // quand `/utilisateur/:id/billets` concorde
+          // quand `/utilisateur/:id/posts` concorde
```
Comment (posva) [inline docs/fr/guide/essentials/nested-routes.md]: it should be the `path` right under that should be adapted to _billets_

**Review by posva — CHANGES_REQUESTED** [review]
Can you also remove the changes

File: docs/fr/guide/essentials/nested-routes.md (hunk 1 above)
Comment (ugocottin) [reply docs/fr/guide/essentials/nested-routes.md]: I did this way to match the english documentation

File: docs/fr/guide/essentials/nested-routes.md [hunk 3]
```diff
@@ -57,13 +57,13 @@ const router = new VueRouter({
...
           path: 'posts',
```
Comment (posva) [inline docs/fr/guide/essentials/nested-routes.md]:
```suggestion
          // quand `/utilisateur/:id/billets` concorde
          path: 'billets',
```

File: docs/fr/guide/essentials/nested-routes.md [hunk 4]
```diff
@@ -87,7 +87,7 @@ const router = new VueRouter({
       children: [
         // `UserProfile` va être rendu à l'intérieur du `<router-view>` de `User`
```
Comment (posva) [inline docs/fr/guide/essentials/nested-routes.md]:
```suggestion
        // `UserHome` va être rendu à l'intérieur du `<router-view>` de `User`
```

File: docs/fr/guide/essentials/nested-routes.md [hunk 5]
```diff
@@ -57,13 +57,13 @@ const router = new VueRouter({
       children: [
         {
           // `UserProfile` va être rendu à l'intérieur du `<router-view>` de `User`
-          // quand `/utilisateur/:id/profil` concorde
+          // quand `/utilisateur/:id/profile` concorde
           path: 'profile',
```
Comment (posva) [inline docs/fr/guide/essentials/nested-routes.md]:
```suggestion
          // quand `/utilisateur/:id/profil` concorde
          path: 'profil',
```

**Review by posva — CHANGES_REQUESTED** [review]
Thanks for noting this! I just need to commit the few changes I said and I will merge it

**Pull request merged**